src/domain/event/EventSummaryPage.tsx

```diff
@@ -73,7 +73,7 @@ const EventSummaryPage: React.FC = () => {
   const isInternalEnrolment = enrolmentType === EnrolmentType.Internal;
 
   const occurrences =
-    (eventData?.event?.pEvent?.occurrences.edges.map(
+    (eventData?.event?.pEvent?.occurrences?.edges.map(
```

Comment:
I don't know when this has changed. I thought it has always been so that occurrences can be null.